applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_phi_pop_layers/MaterialParameters.json

```diff
@@ -0,0 +1,136 @@
+{
+    "properties": [
+        {
+            "properties_id": 1,
+            "model_part_name": "PorousDomain.Soil1",
+            "Material": {
+                "constitutive_law": {
+                    "name": "GeoLinearElasticPlaneStrain2DLaw"
+                },
+                "Variables": {
+                    "BIOT_COEFFICIENT": 1.0,
+                    "DENSITY_SOLID": 2548.419975535168,
+                    "DENSITY_WATER": 1019.368,
+                    "POROSITY": 0.3333333333333333,
+                    "BULK_MODULUS_SOLID": 20000000000.0,
+                    "BULK_MODULUS_FLUID": 2e-30,
+                    "PERMEABILITY_XX": 4.5e-30,
+                    "PERMEABILITY_YY": 4.5e-30,
+                    "PERMEABILITY_XY": 0.0,
+                    "DYNAMIC_VISCOSITY": 9810.0,
+                    "RETENTION_LAW": "SaturatedLaw",
+                    "SATURATED_SATURATION": 1.0,
+                    "MINIMUM_RELATIVE_PERMEABILITY": 0.0001,
+                    "YOUNG_MODULUS": 10000.0,
+                    "POISSON_RATIO": 0.3,
+                    "IGNORE_UNDRAINED": true,
+                    "K0_MAIN_DIRECTION": 1,
+                    "POP": 1400,
+                    "POISSON_UNLOADING_RELOADING": 0.2,
+                    "UDSM_NAME"                :  "../MohrCoulomb64.dll",
```

Comment:
Based on the kind of constitutive law (`GeoLinearElasticPlaneStrain2DLaw`), I don't think we need to specify which UDSM DLL we'd like to use.

Reply:
removed.